soap/README_Soap.md

```diff
@@ -0,0 +1,142 @@
+
+## What is LoopBack?
+
+[LoopBack](http://loopback.io) is an open source Node.js API framework from [StrongLoop](http://www.strongloop.com). 
+It is built on top of Express optimized for mobile, web, and other devices.  LoopBack makes 
+it really easy and productive for developers to define, build and consume APIs. Define data 
+models, connect to multiple data sources, write business logic in Node.js, glue on top of your
+existing services and data, and consume using JS, iOS & Android SDKs.
+
+## What is Web Service/SOAP/WSDL?
+
+Web services is a technology which lets applications to talk to each other independent 
+of platform and language. A web service is a software interface which describes a collection 
+of operations that can be invoked over the network through standardized XML messaging using 
+SOAP(Simple Object Access Protocol). WSDL(Web Services Description Language) is an 
+XML formatted document which describes the Web Service endpoint, bindings, operations 
+and schema.
+
+## LoopBack + SOAP = REST API
+
+![loopback soap](images/loopback-soap-integration.png) 
+
+In many Enterprises, Web Services are still important and the way to access these
+Web Services is still via SOAP. SOAP is fairly heavy weight, and working with XML-based SOAP 
+payloads in Node.js is not easy.  It’s much easier to use JSON and to wrap or mediate a SOAP 
+service and expose it as a REST API. To support API design first approach, 'lb soap' feature
+supports generation of JSON models and REST APIs for WSDL/SOAP operations. These operations 
+can invoke Web Service without user writing any client code.
+
+## Scaffolding a LoopBack application from WSDL
+
+Before we start, please make sure you have loopback-cli installed:
+
+```sh
+npm install -g loopback-cli
+```
+For more information, see https://github.com/strongloop/loopback-cli
+
+### Create a loopback application
+
+The first step is to create a blank loopback application. For e.g soap-demo
+
+```sh
+lb app
+```
+Select 3.x or 2.x Loopback version. When prompted 'What kind of application do you have in mind?', 
+select 'empty-server' for this demo purpose.
+
+![lb soap](images/loopback-app.png)
+
+### Generate APIs from WSDL
+
+Now let's try to generate APIs from WSDL document. Note: There is an enhancement planned to generate 
+APIs given SOAP Web Service data source.
+
+```sh
+cd soap-demo
+lb soap
+```
+When prompted, provide an url or local file path to the WSDL. In this demo, we will use WSDL from an 
+externally available Web Service:
+
+http://www.webservicex.net/periodictable.asmx?WSDL
+
+The generator loads the WSDL and discovers services defined in the WSDL. It then prompts you to 
+select the service from a list of services.
+
+![lb soap:](images/loopback-soap-service.png)
+
+Once a 'service' is selected, it will discover and list bindings defined for the selected service. 
+Select a binding.
+
+![slc loopback:swagger](images/loopback-soap-binding.png)
```

Comment:
There are a few references to `slc loopback:swagger`. I think it should be `lb soap` now.